flink-streaming-connectors/flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/common/container/RedisContainer.java

```diff
@@ -68,12 +68,12 @@ public RedisContainer(final JedisSentinelPool sentinelPool) {
 	 */
```

Comment:
The `RedisContainer` class contains fields which are not serializable. The `jedisPool` and the `jedisSentinelPool`.